requirements-gpu.txt

```diff
@@ -0,0 +1,5 @@
+torch
+transformers>=4.38
+threadpoolctl
+lm-eval>=0.4.2,<0.5
+auto-gptq>=0.7.1
```

Comment:
the above 4 should be in requirement

Reply:
> the above 4 should be in requirement

All platforms will install requirement.txt, we can't add incompatible packages

Reply:
The basic requirement must support tuning on GPU, otherwise, the user experience will suffer. We may release a whl for HPU.